dist/receiver.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Ooyala Receiver</title>
+
+    <script type="text/javascript" src="//www.gstatic.com/cast/sdk/libs/receiver/2.0.0/cast_receiver.js"></script>
+    <script type="text/javascript" src="//player.ooyala.com/core/e48c9b51282d406f957c666149763424?plugins=bm"></script>    
```

Comment:
How is e48c9b51282d406f957c666149763424 used? is this hard-coded or generated? How customer should replace this with their playerID?